accelerator/core/core.c

```diff
@@ -616,12 +616,15 @@ status_t push_txn_to_buffer(const ta_cache_t* const cache, hash8019_array_p raw_
     goto done;
   }
 
-  // TODO We push only one transaction raw trits into list, we need to solve this in future.
-  ret = cache_set(uuid, UUID_STR_LEN - 1, hash_array_at(raw_txn_flex_trit_array, 0),
-                  NUM_FLEX_TRITS_SERIALIZED_TRANSACTION, cache->timeout);
-  if (ret) {
-    ta_log_error("%s\n", ta_error_to_string(ret));
-    goto done;
+  const int len = hash_array_len(raw_txn_flex_trit_array);
+  // We assume all the transactions in a single hash_array would be in the same bundle.
```

Comment:
Should such assumption be clarified in advance?

Reply:
We buffer transaction only when `ta_send_trytes()` is failed which means the implemetation implies `ta_send_trytes()` can send only one bundle each time.

Reply:
Explain within the comments.